tests/cases/standalone/common/copy/copy_from_fs_parquet.result

```diff
@@ -93,22 +93,26 @@ select count(*) from without_limit_rows;
 | 4        |
 +----------+
 
-CREATE TABLE with_limit_rows(host string, cpu double, memory double, ts timestamp time index);
+CREATE TABLE with_limit_rows_segment(host string, cpu double, memory double, ts timestamp time index);
 
 Affected Rows: 0
 
-Copy with_limit_rows FROM '/tmp/demo/export/parquet_files/' WITH (MAX_INSERT_ROWS = 2);
+Copy with_limit_rows_segment FROM '/tmp/demo/export/parquet_files/' LIMIT 2;
 
 Affected Rows: 2
 
-select count(*) from with_limit_rows;
+select count(*) from with_limit_rows_segment;
 
 +----------+
 | COUNT(*) |
 +----------+
 | 2        |
 +----------+
 
+Copy with_limit_rows_segment FROM '/tmp/demo/export/parquet_files/' LIMIT -1;
+
+Error: 2000(InvalidSyntax), Unexpected token while parsing SQL statement: Copy with_limit_rows_segment FROM '/tmp/demo/export/parquet_files/' LIMIT -1;, expected: 'maximum rows', found: 1: sql parser error: Expected literal int, found: - at Line: 1, Column 75
```

Comment:
As you can see, the error comes from the sql parser itself as it can't parse `-1`, not from our codes.